### PR TITLE
ci(): guides follow up

### LINKED ISSUE #8378: task(): new bug report links are broken
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.2.4

### In What environments are you experiencing the problem?

Firefox, Safari, Node.js

### Node Version (if applicable)

18.11.0

### Link To Reproduction

some link

### Steps To Reproduce

1. a
2. v
3.

### Expected Behavior

links should work in the template

### Actual Behavior

they don't

### Error Message & Stack Trace

_No response_

### PR DESCRIPTION
**Pull request by ShaMan123**
Fix left overs from #8189 
closes #8378 

- typos
- links - the new form templates don't support relative links yet
- add feature request template
- `.github/workflows/npmpublish.yml`: update bug report action to run on master always

### CODE REVIEW

**Review by ShaMan123 — COMMENTED** [review]
READY

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ body:
         Before you do, please ensure you are filing the issue in the right
         place. 
 
-        * If you have a question or if an issue is not right for what you mean
+        If you have a question or if an issue is not right for what you mean
```
Comment (melchiar) [inline .github/ISSUE_TEMPLATE/bug_report.yml:13]:
```suggestion
        If you have a question or if an issue is not right for what you mean
```
The wording here is a bit awkward. Maybe something like "If you have a question, or would like to start a discussion that's not does not fit the issue criteria, see (link)"

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      value: >
+        Thank you for wanting to contribute to this repository. 
+
+        Before you do, please ensure you are filing the ticket in the right
+        place. 
+
+        If you have a question use
+        [Discussions](https://github.com/fabricjs/fabric.js/discussions).
+
+        If you are up for PRing see [Pull Requests](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#-pull-requests).
```
Comment (melchiar) [inline .github/ISSUE_TEMPLATE/feature_request.yml:16]:
```suggestion
        If you are interested in submitted a PR see [Pull Requests](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#-pull-requests).         
```

We should probably stick to clear English for the benefit of google translate users

**Comment by melchiar** [comment]
looks good, just a few small suggestions.

**Review by melchiar — APPROVED** [review]

**Pull request merged**